test/cases/fixes/noInferableTypes/variableDeclarations/expected.ts

```diff
@@ -47,8 +47,8 @@
 	let letInferableNumberOrRegExps = [0, /./];
 
 	// Non-inferable const multi-type arrays
```

Comment:
[Docs] These are inferable now! 

```suggestion
	// Inferable const multi-type arrays
```

Reply:
Ah, right. Thanks 😄